### DISCUSSION TITLE
Welcome to gokrazy Discussions!

### DISCUSSION THREAD

**Discussion by stapelberg** [gokrazy/gokrazy#136] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by damdo** [comment]
Hey, I'll start.

Nice to meet you all I'm Dam (@damdo here on GitHub) 👋 

I really look forward to see a community of enthusiasts grow around gokrazy and I'm excited to share ideas and see what my fellow _gokrazists_ (? we need to find a word to describe ourselves 😬, what do y'all propose? see https://github.com/gokrazy/gokrazy/discussions/138) use/plan to use gokrazy for.

Thanks for creating a GitHub Discussions space for gokrazy, @stapelberg !

**Comment by oliverpool** [comment]
Hi,

Nice to meet you. I'm Olivier (a french guy living in Germany :fr: :de: )

Thank @stapelberg for this awesome project. I contributed to the support of the Raspeberry Pi Model B.
I finally wrote a blog post documenting my very first steps with gokrazy :smile: https://log.pfad.fr/2022/go-ing-krazy-raspberry-pi-model-b/

I am also eager to know what all of you use gokrazy for! (I might document my own use in another blog post ;)

**Reply by damdo** [reply]
Nice to meet you @oliverpool, looking forward to hack gokrazy together!

**Reply by damdo** [reply]
BTW, I'm using https://github.com/gokrazy-community/kernel-rpi-os-32 for a couple of gokrazy 24/7 instances and it works great!
